demo_hybrid_search.py

```
@@ -0,0 +1,245 @@
+#!/usr/bin/env python3
+"""
+Demonstrate how hybrid search combines Neo4j + Qdrant for superior results.
+"""
+import asyncio
+import logging
+from unified_ingestion import UnifiedIngestionPipeline, IngestionConfig
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+async def demo_hybrid_search_superiority():
```

Ruff (0.12.2)

`demo_hybrid_search_superiority` is too complex (16 > 10)

(C901)